### PR TITLE
[DOC] Add header to cookie secret paragraph

### PR DESCRIPTION
**Pull request by koriit-kontakt**
## Description
Added a header to the paragraph about cookie secret generation. It should be easier to find it given that it should be displayed in the TOC.

### CODE REVIEW

File: docs/docs/configuration/overview.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ title: Overview
 
 `oauth2-proxy` can be configured via [config file](#config-file), [command line options](#command-line-options) or [environment variables](#environment-variables).
 
+### Cookie Secret
```
Comment (JoelSpeed) [inline docs/docs/configuration/overview.md:8]: I like the idea, maybe `Generating a Cookie Secret`?

Could you perhaps also add this to the v7/v6 versioned docs as well? I think this is worth back porting

Comment (koriit-kontakt) [reply docs/docs/configuration/overview.md:8]: Sure.

Comment (koriit-kontakt) [reply docs/docs/configuration/overview.md]: @JoelSpeed done.

**Review by JoelSpeed — APPROVED** [review]
Looks great! Thanks

**Pull request merged**